deployments/helm/hephaestus/templates/controller/cronjob.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: image-build-gc
+spec:
+  schedule: "* * * * *"
+  jobTemplate:
+    spec:
+      template:
+        spec:
```

Comment:
We'll want to make sure we support the same configurables and defaults as the manager deployment: 
https://github.com/dominodatalab/hephaestus/blob/main/deployments/helm/hephaestus/templates/controller/deployment.yaml#L36

* pod labels, annotations
* pod security context
* container security context
* image pull policy (`imagePullPolicy: {{ .Values.controller.manager.image.pullPolicy }}`)
* `extraEnvVars`
* we should consider having some `resources` definition, even if it's quite low
* nodeSelector
* affinity
* tolerations
* priority class

For the majority of these you can use the same Helm values as the manager. Resources might be one to keep separate.